pkg/webhook/certs.go

```diff
@@ -333,16 +342,16 @@ func lookaheadTime() time.Time {
 	return time.Now().Add(90 * 24 * time.Hour)
 }
 
-func (cr *certRotator) validServerCert(caCert, cert, key []byte) bool {
-	valid, err := validCert(caCert, cert, key, cr.dnsName, lookaheadTime())
+func (cr *CertRotator) validServerCert(caCert, cert, key []byte) bool {
+	valid, err := validCert(caCert, cert, key, cr.DNSName, lookaheadTime())
 	if err != nil {
 		return false
 	}
 	return valid
 }
 
-func (cr *certRotator) validCACert(cert, key []byte) bool {
-	valid, err := validCert(cert, cert, key, cr.caName, lookaheadTime())
+func (cr *CertRotator) validCACert(cert, key []byte) bool {
+	valid, err := validCert(cert, cert, key, cr.CaName, lookaheadTime())
```

Comment:
Apparently the go style guide says this should be CAName, sorry I used the wrong casing :/

Reply:
or caName

Reply:
Thanks for catching this!